certbot/main.py

```diff
@@ -382,6 +382,28 @@ def register(config, unused_plugins):
     # exist or not.
```

Comment:
The `register` function here is getting a little long and should probably be broken up into smaller functions. While I think this should definitely at some point, this can be done in another PR if you prefer.